### PR TITLE
Do URL resolution in Analyzer. Make Analyzer#parse private.

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by rictic** [comment]
Please add an integration test to test/static/analysis/

**Comment by justinfagnani** [comment]
PTAL

File: src/test/static/analysis/bower_packages/elements.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (rictic) [inline src/test/static/analysis/bower_packages/elements.json]: Need to add `src/test/static/analysis/bower_packages/bower_components`

**Comment by rictic** [comment]
LGTM

**Pull request merged**